### PR TITLE
UI/home page

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- [UI home Page](https://trello.com/c/iXCJPfLQ/22-home-page)

## Todo
- UI home page
- UI story view

## Video

https://github.com/luke-nguyen-goldenowl/join_me/assets/152830147/8597019a-4892-430a-8ab1-30e7c6c975e5

https://github.com/luke-nguyen-goldenowl/join_me/assets/152830147/53fdb765-8c17-4fa5-b9f4-aee523fbeaec

### CODE REVIEW

File: lib/src/features/home/story/widget/story_widget.dart [hunk 1]
```diff
@@ -0,0 +1,238 @@
+import 'package:flutter/material.dart';
+import 'package:myapp/src/features/home/data/users.dart';
+import 'package:myapp/src/features/home/model/story.dart';
+import 'package:myapp/src/features/home/model/user.dart';
+import 'package:myapp/src/features/home/story/widget/profile_widget.dart';
+import 'package:myapp/src/router/coordinator.dart';
+import 'package:myapp/src/theme/colors.dart';
+import 'package:story_view/story_view.dart';
+
+class StoryWidget extends StatefulWidget {
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/home/story/widget/story_widget.dart]: Please use Cubit with StateLessWidget

File: lib/src/features/home/widget/story_item.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import 'package:myapp/src/features/home/logic/story_item_state.dart';
+import 'package:myapp/src/router/coordinator.dart';
+import 'package:myapp/src/theme/colors.dart';
+
+class StoryItem extends StatelessWidget {
+  const StoryItem({super.key, required this.id});
+  final String id;
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<StoryItemBloc, StoryItemState>(
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/home/widget/story_item.dart:15]: add buildWhen

**Comment by Keith-Go** [comment]
resolved

**Comment by luke-nguyen-goldenowl** [comment]
Add buildwhen  and remove controller from state file.
After fixing, you can merge

**Pull request merged**